app/ts/components/TransactionPage/TransactionDetails.tsx

```diff
@@ -155,28 +154,7 @@ type TokenAmountProps = {
 }
 
 const TokenAmount = ({ receipt, addFavoriteStore }: TokenAmountProps) => {
-	const { query, tokenAddress } = useTokenQuery()
-
-	if (receipt.to === null) return <></>
-	tokenAddress.value = receipt.to
-
-	const txLog = extractTransferLogFromSender(receipt)
-	if (txLog === null) return <></>
-
-	const tokenValue = extractArgValue<bigint>(txLog, 'value')
-	if (tokenValue === null) return <></>
-
-	switch (query.value.state) {
-		case 'inactive':
-			return <></>
-		case 'pending':
-			return <InfoPending />
-		case 'rejected':
-			return <InfoError displayText='Failed to get token amount' message={query.value.error.message} />
-		case 'resolved':
-			const { decimals, symbol } = query.value.value
-			const amount = formatUnits(tokenValue, decimals)
-			addFavoriteStore.value = { ...addFavoriteStore.peek(), amount, token: query.value.value }
-			return <Info label='Amount' value={`${amount} ${symbol}`} />
-	}
+	// TODO: implement amount query
+	console.log('token amount', receipt, addFavoriteStore)
+	return <></>
```

Comment:
I feel this should be done before this is merged?

Reply:
updated the current PR to include the changes